includes/class-wcsr-resource.php

```diff
@@ -202,8 +202,8 @@ public function get_days_active( $from_timestamp, $to_timestamp = null ) {
 		}
 
 		// Find all the activation and deactivation timestamps between the given timestamps
-		$activation_times   = self::get_timestamps_between( $this->get_activation_timestamps(), $from_timestamp, $to_timestamp );
-		$deactivation_times = self::get_timestamps_between( $this->get_deactivation_timestamps(), $from_timestamp, $to_timestamp );
+		$activation_times   = wcsr_get_timestamps_between( $this->get_activation_timestamps(), $from_timestamp, $to_timestamp );
+		$deactivation_times = wcsr_get_timestamps_between( $this->get_deactivation_timestamps(), $from_timestamp, $to_timestamp );
 
 		// if the first activation date is after the first deactivation date, make sure we prepend the start timestamp to act as the first "activated" date for the resource
 		if ( ! isset( $activation_times[0] ) || ( isset( $deactivation_times[0] ) && $activation_times[0] > $deactivation_times[0] ) ) {
```

Comment:
The first stored timestamp on `$activation_times` can actually be located at `$activation_times[1]` i.e. `wcsr_get_timestamps_between()` can return an array that looks like 

```
array (
  1 => '1519276153',
)
```


At the next if statement this can mean the difference between days active being reported as 17 or (in the case of recent renewal of mine) 45 😅